### PR TITLE
Add insecure and secure virus total entries

### PR DESCRIPTION
**Pull request by ErebusZ**

### CODE REVIEW

File: MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/description.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Malware analysis by multiple Antiviruses using VirusTotal has report .
```
Comment (3asm) [inline MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/description.md:1]: This is not even English.

File: MOBILE_CLIENT/COMMON/_HIGH/INSECURE_VIRUSTOTAL_SCAN/description.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Malware analysis by multiple Antiviruses using VirusTotal MD5 hash based search.
+
+An insecure VirusTotal scan indicates the potential presence of harmful elements such as viruses, malware, and malicious content, this opens up the door to cyberattacks.
+
+Moreover, associating with flagged entities can result in reputational damage, low search engine rankings and regulatory compliance issues.
```
Comment (3asm) [inline MOBILE_CLIENT/COMMON/_HIGH/INSECURE_VIRUSTOTAL_SCAN/description.md:5]: Needs reworking.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/meta.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-  "risk_rating": "info",
+  "risk_rating": "secure",
   "short_description": "VirusTotal Malware analysis.",
   "references": {
     "Virustotal": "https://www.virustotal.com/"
   },
-  "title": "Virustotal malware analysis (MD5 based search)",
+  "title": "Secure Virustotal malware analysis (MD5 based search)",
   "privacy_issue": true,
   "security_issue": true
```
Comment (deadly-panda) [inline MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/meta.json:9]: The `privacy_issue` & `security_issue` should be set to false in the secure case, no?

File: MOBILE_CLIENT/COMMON/_HIGH/INSECURE_VIRUSTOTAL_SCAN/recommendation.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+If you discover that one of the assets associated with your application has a bad reputation or is flagged as malicious, it's important to take action to address the issue and ensure the security and reliability of your application.
+Actions you need to take will depend on the nature of the asset, your control over it, and the severity of the reported issue.
+
+Here are some recommendations on what to do in such a situation:
+
+ * Determine why the asset was flagged as malicious. It could be due to various factors, such as hosting malicious content, containing malware, spam, or being associated with a history of cyberattacks. Investigate the specifics of the problem.
+
+ * Consider isolating it from your primary systems. You may want to temporarily disable or quarantine the asset to prevent further potential issues while you investigate.
+
+ * If the asset is associated with a service you control, take steps to patch and secure it. This might involve updating software, implementing stronger security measures, and ensuring it is not used for malicious purposes.
```
Comment (deadly-panda) [inline MOBILE_CLIENT/COMMON/_HIGH/INSECURE_VIRUSTOTAL_SCAN/recommendation.md:10]: This feels more like the reputation KB entry.

Comment (ErebusZ) [reply MOBILE_CLIENT/COMMON/_HIGH/INSECURE_VIRUSTOTAL_SCAN/recommendation.md:10]: they are basically the same

File: MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/description.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Performing malware analysis through multiple antivirus scans using VirusTotal's MD5 hash-based search.
```
Comment (BlueSquare1) [inline MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/description.md:1]: This should tell the user that the results of the analysis are negative

File: MOBILE_CLIENT/COMMON/_HIGH/INSECURE_VIRUSTOTAL_SCAN/meta.json [hunk 6]
```diff
@@ -1,10 +1,10 @@
 {
-  "risk_rating": "info",
+  "risk_rating": "high",
   "short_description": "VirusTotal Malware analysis.",
   "references": {
     "Virustotal": "https://www.virustotal.com/"
   },
-  "title": "Virustotal malware analysis (MD5 based search)",
+  "title": "Vulnerable Virustotal malware analysis (MD5 based search)",
```
Comment (BlueSquare1) [inline MOBILE_CLIENT/COMMON/_HIGH/INSECURE_VIRUSTOTAL_SCAN/meta.json:7]: "Vulnerable Virustotal malware analysis" doesn't make much sense

**Review by BlueSquare1 — CHANGES_REQUESTED** [review]

File: MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/description.md (hunk 5 above)
Comment (amine3) [reply MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/description.md:1]: +1

**Review by amine3 — DISMISSED** [review]

**Comment by 3asm** [comment]
Ping.

**Review by 3asm — APPROVED** [review]

**Pull request merged**